book/04-git-server/sections/setting-up-server.asc

```diff
@@ -89,15 +89,15 @@ You should note that currently all these users can also log into the server and
 If you want to restrict that, you will have to change the shell to something else in the `/etc/passwd` file.
 
 You can easily restrict the `git` user account to only Git-related activities with a limited shell tool called `git-shell` that comes with Git.
-If you set this as the `git` user account's login shell, then that  account can't have normal shell access to your server.
-To use this, specify `git-shell` instead of bash or csh for that account's login shell.
+If you set this as the `git` user account's login shell, then that account can't have normal shell access to your server.
+To use this, specify `git-shell` instead of `bash` or `csh` for that account's login shell.
 To do so, you must first add the full pathname of the `git-shell` command to `/etc/shells` if it's not already there:
 
 [source,console]
 ----
-$ cat /etc/shells   # see if `git-shell` is already in there.  If not...
-$ which git-shell   # make sure git-shell is installed on your system.
-$ sudo -e /etc/shells  # and add the path to git-shell from last command
+$ cat /etc/shells   # see if `git-shell` is already in there. If not...
+$ which git-shell   # make sure `git-shell` is installed on your system.
+$ sudo -e /etc/shells  # and add the path to `git-shell` from last command
```

Comment:
This is already inside a monospace block, so the backticks might interfere with copy-pasting this into a console. Let's revert these two lines.

Reply:
Backticks already been set in line above. Removed from all three lines.